src/Main.java

```diff
@@ -18,11 +18,45 @@ public static void main(String[] args) {
         TicketService ticketService = new TicketService();
 
         // Retrieve a ticket by ID
-        Ticket ticket = ticketService.getTicketById("ID5");
+        Ticket ticket = ticketService.getTicketById(String.valueOf(5));
         if (ticket != null) {
             System.out.println("Found Ticket: " + ticket);
         } else {
             System.out.println("Ticket not found.");
         }
+
+
+        // Modify allowed fields using TicketModifiable interface methods
+        fullTicket.setTime(System.currentTimeMillis() + 10000); // Update time
+        fullTicket.setStadiumSector('B'); // Update stadium sector
+
+        // Display all values using Valuable interface method
+        System.out.println("Updated Ticket Values: " + fullTicket.getAllValues());
+
+        Shareable shareByPhone = new ShareByPhone(ticket);
+        Shareable shareByEmail = new ShareByEmail(ticket);
+
+        // Sharing the ticket by different methods
+        shareByPhone.share(); // Calls the phone sharing method
+        shareByEmail.share(); // Calls the email sharing method
```

Comment:
I will explain this comment below.
 But for now, think about this: if I want to send another ticket, do I need to create another `SendByEmail `class?